data/templates/create_jbod.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+
+# Copyright 2015, EMC, Inc.
+
+'''
+This script is to delete RAID and create JBOD for each physical disk of a virtual disk with RAID
+'''
+import subprocess
+import argparse
+import json
+import re
+import time
+from os import path as path
+
+ARG_PARSER = argparse.ArgumentParser(description=
```

Comment:
As name convention, `ARG_PARSER` is not constant, so it should not be all in upper cases.